lib/src/main/java/dataframe/transformation/base/PrimitiveArrayTransformation.java

```diff
@@ -3,10 +3,13 @@
 public abstract class PrimitiveArrayTransformation implements PrimitiveTransformation {
 
     public Object apply(Object o) {
-        if(!(o instanceof Object[])) {
+        if (o.toString().trim().isEmpty()) {
+            return "";
```

Comment:
Why is this required? The object must be an instance of array which should be the first condition here